src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,634 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import java.util.Arrays;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Arrays'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)